src/main/java/org/fcrepo/spec/testsuite/crud/Container.java

```diff
@@ -158,9 +170,15 @@ public void ldpcContainmentTriples() {
         confirmPresenceOrAbsenceOfTripleInResponse(resP, getLocation(container), LDP_CONTAINS_PREDICATE,
                                                    getLocation(containerChild), true);
 
-        // Verify absence of unexpected triple in response body
-        confirmPresenceOrAbsenceOfTripleInResponse(resP, getLocation(container), LDP_CONTAINS_PREDICATE,
-                                                   getLocation(directMember), false);
+        if (directContainerTripleCheck) {
+            // Verify absence of unexpected triple in response body
+            confirmPresenceOrAbsenceOfTripleInResponse(resP, getLocation(container), LDP_CONTAINS_PREDICATE,
+                                                       getLocation(directMember), false);
+        }
+
+        // Check for presence of the dcterms:title triple.
+        confirmPresenceOrAbsenceOfPredicateInResponse(resP, getLocation(container), "http://purl.org/dc/terms/title",
```

Comment:
In my testing running against fcrepo4 version 5.1.0, it fails here. This is because the test suite creates the container `container` with the [BASIC_CONTAINER_BODY](https://github.com/fcrepo/Fedora-API-Test-Suite/blob/master/src/main/java/org/fcrepo/spec/testsuite/Constants.java#L32), which has a dc:title in the body.  I'm not clear on what feature you're testing with this check here.

Reply:
Altogether possible I'm missing some variable in the testing here, though.

Reply:
Sounds like Fedora has the right semantics to pass my predicate check, but I'll have a look to see if I can explain that lack of namespace expansion. I thought that was automatic, but it may not be. Can you determine if the Fedora response includes the dc namespace declaration, as usual?

Reply:
Oh, nevermind. That confirmPresenceOrAbsenceOfPredicateInResponse call is there to make sure that the response *only* includes containment triples and nothing else, which is what I expected by using "Prefer ... omit=...PreferMinimalContainer". It sounds like Fedora is not honoring the omit instruction. I'm going to check back in LDP specs to make sure I have the semantics right.

Reply:
Ah, I'll look into that as well from the fcrepo side of things.

Reply:
Okay, so this section of LDP defines the set of PreferMinimalContainer triples as the set of triples that would be present if the container were empty, i.e. dc:title. So dc:title should not be there if I ask the server to omit PreferMinimalContainer.

Reply:
link: https://www.w3.org/2012/ldp/hg/ldp.html#dfn-minimal-container-triples

Reply:
Actually, I wonder if this is a configuration of fcrepo issue on my end.  I'm going to try one thing and get back to you.

Reply:
I need to look into this a little more (later), because my read of that is that it would include any triples that were not containment or membership triples, so wouldn't dc:title be present still?  Also, I think the PreferMinimalContainer needs to be in the `include` part of the prefer header - or rather fcrepo is expecting it to be in the `include` and not `omit` (this could be wrong - not sure).  Although, honestly, this is all a bit fuzzy to me and I'm still sorting out my thoughts.

Reply:
But I can see that perhaps if that flag is in the omit, one could expect to not get the minimal containment triples, but I'm not sure fcrepo would do that.  (and it's possible that there's a bug here in fcrepo).

Reply:
Yes, in the older version of this test did not check for "minimal" triples. That was a new check that I added when I realized the test wasn't quite comprehensive.

Reply:
It's a little complicated because these tests were using "include" before in order to test the absence of particular triples, but include doesn't guarantee absence, just inclusion. So I flipped these tests to use "omit".